sphinx_gallery/gen_rst.py

```diff
@@ -498,7 +499,8 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     ref_fname = example_file.replace(os.path.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot'):
+    fname_pattern = gallery_conf.get('fname_pattern', 'plot_*')
+    if os.path.join(src_dir, fname) not in glob.glob(fname_pattern):
```

Comment:
https://docs.python.org/2/howto/regex.html#the-backslash-plague might be your answer :) 

I'm not too familiar with regexp so I'm already biased against it (so also others: https://xkcd.com/1171/ ;-). But I can live with something like `fnmatch.translate` for myself. To avoid switching back and forth between the two versions, it would be good to know what the general consensus is.